### ISSUE TITLE
Getting compilation Error : Cannot find module 'lotion/lib/json.js'

### ISSUE THREAD

**Issue by Debananda** [mappum/coins#12]
Error: 
```
node validatorNode.js /home/debananda/Workspace/lotion-coin/validator/node_modules/supercop.js/lib.js:97
      throw ex;
      ^

Error: Cannot find module 'lotion/lib/json.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
```

"dependencies": {
    "coins": "^3.1.2",
    "lotion": "^0.1.19",
    "lotion-connect": "^0.1.2"
  }

Validator Code : 
```
let coins = require("coins");
let lotion = require("lotion");

let app = lotion({
  devMode: true,
  genesis: "./genesis.json",
  tendermintPort: 30090,
  p2pPort: 30092,
  logTendermint: false,
  keys: "validatorKey.json",
  peers: ["localhost:30092"]
});

app.use(
  coins({
    initialBalances: {
      judd: 10,
      matt: 10
    }, ...
```

**Comment by yoozzeek** [comment]
You need change  sigHash.js file:
line:2` let stableStringify = require('json-stable-stringify')`
line:21 `let txString = stableStringify(tx)`

A don't know it is right way or not, but it's worked and help for me
And, not use `npx coins`, use  `npm bin.js`

**Comment by mappum** [comment]
Fixed in 54d5d900684f3c81a3f29c2724b0f5bfa4493194
